packages/client/src/client/notifications.finsemble.ts

```diff
@@ -43,3 +43,7 @@ export function registerCreditQuoteNotifications() {
 export function unregisterCreditQuoteNotifications() {
   // no-op
 }
+
```

Comment:
this file needs updating now .. _really_ need that type for the "variant modules"